test/e2e/specs/config.js

```diff
@@ -0,0 +1,12 @@
+export const MINUTE_MS = 60000;
+
+export const phrasesList = {
+  emerynet: {
+    interNetwork: 'Agoric Emerynet',
+    isLocal: false,
+  },
+  local: {
+    interNetwork: 'Local Network',
+    isLocal: true,
+  },
```

Comment:
There is a file called `test.utils.js`. Perhaps these declarations can be placed there?